lib/utils/jsontools.go

```diff
@@ -20,7 +20,10 @@ import (
 	"bytes"
 	"unicode"
 
+	"github.com/gravitational/trace"
+
 	"github.com/ghodss/yaml"
+	"github.com/json-iterator/go"
```

Comment:
dependency is not vendored

Reply:
I was actually surprised by this as well, turns out it's already vendored.